packages/appeal-reply-service-api/src/controllers/reply.js

```diff
@@ -62,6 +62,28 @@ module.exports = {
     }
   },
 
+  async getByAppeal(req, res) {
+    const appealId = req.params.id;
+    logger.debug(`Retrieving reply by appeal ID ${appealId} ...`);
+    try {
+      await mongodb
+        .get()
+        .collection(dbId)
+        .findOne({ 'reply.appealId': { $eq: appealId } })
+        .then((doc) => {
+          logger.debug(`Reply for appeal ${appealId} retrieved`);
+          res.status(200).send(doc.reply);
+        })
+        .catch((err) => {
+          logger.warn(`Could not find reply for appeal ${appealId}\n${err}`);
+          res.status(404).send(null);
+        });
+    } catch (err) {
+      logger.error(`Problem retrieving reply for appeal ${appealId}\n${err}`);
```

Comment:
`logger.error({appealId, err}, 'Problem retrieving reply for appeal');`

or you will lose the correlation id

Reply:
Good catch, will add this in